docs/user-guide/slicing.ipynb

```diff
@@ -217,6 +226,174 @@
+    "        'y': sc.Variable(['y'], values=np.arange(4.0), unit=sc.units.m)})\n",
+    "sc.show(da)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets say we wish to slice the `DataArray` where y coord is exactly at the point 1.0 meters expressed as a variable. You will notice that the data item in the output have no y coordinate dependency after the point slicing operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "point_value = 0.0 * sc.units.m\n",
+    "da['y', point_value] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our y-coordinate is represents point data rather than bin edges for the dataset data. This means that when slicing with a point value, the value needs to exactly match a coordinate value. If it does not, an `IndexError` is raised."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "point_value = 0.1 * sc.units.m # No y coordinate value at this point\n",
+    "try:\n",
+    "    d['y', point_value]\n",
+    "except IndexError as e:\n",
+    "    print(str(e))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also slice a range just like index slicing above. No exact match to coordinate value of start/stop is required in the case of range slicing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = 0.0 * sc.units.m\n",
+    "stop = 1.1 * sc.units.m\n",
+    "d['y', start:stop]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Value slicing on bin edge coordinates\n",
+    "\n",
+    "As mentioned above, value based slicing is valid for `Datasets` and `DataArrays` We construct a simple `DataArray` to illustrate the working. In this `DataArray` the y-coordinate is bin edges."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = sc.DataArray(data = sc.Variable(dims=['x', 'y'], values=np.arange(8.0).reshape(2, 4)),\n",
+    "    coords={\n",
+    "        'x': sc.Variable(['x'], values=np.arange(2.0), unit=sc.units.m),\n",
+    "        'y': sc.Variable(['y'], values=np.arange(5.0), unit=sc.units.m)})\n",
+    "sc.show(da)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because we have bin edges in the y dimension, each bin represents a range between the defined edges. Therefore point slicing between coordinate values is a valid operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "point_value = 0.1 * sc.units.m # select between coord values 0 - 1 meter\n",
```

Comment:
I think this comment is misleading. Rather explain (in the text) that slicing without range will find and return the bin that contains the given coord value.